src/App.tsx

```diff
@@ -1,24 +1,86 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, {
+  useContext, useEffect, useState,
+} from 'react';
+import { getTodos } from './api/todos';
+import { AuthContext } from './components/Auth/AuthContext';
+import { ErrorMessage } from './components/ErrorMessage';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { FilterStatus } from './types/FilterStatus';
+import { TempTodo, Todo } from './types/Todo';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [tempTodo, setTempTodo] = useState<TempTodo | null>(null);
+  const [error, setError] = useState('');
```

Comment:
create a enum for errorType, it is not easily maintainable to work with string values